widget/playground/src/components/SupportedWallets/SupportedWallets.tsx

```diff
@@ -0,0 +1,167 @@
+import type { PropTypes } from './SupportedWallets.types';
+import type { WalletTypes } from '@rango-dev/wallets-shared';
+
+import {
+  Button,
+  Checkbox,
+  CloseIcon,
+  Divider,
+  IconButton,
+  Image,
+  ListItemButton,
+  SearchIcon,
+  SelectableCategoryList,
+  TextField,
+  Typography,
+  WalletIcon,
+} from '@rango-dev/ui';
+import React, { useState } from 'react';
+
+import { NOT_FOUND } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+import { StyledButton } from '../FontSelector/FontSelector.styles';
+
+import {
+  HeaderContainer,
+  IconWrapper,
+  SelectButton,
+  WalletDivider,
+  WalletList,
+} from './SupportedWallets.styles';
+
+export function SupportedWallets(props: PropTypes) {
+  const { onBack, configWallets, allWallets } = props;
+  const [category, setCategory] = useState<string>('ALL');
+  const [searchValue, setSearchValue] = useState('');
+  const [selectedWallets, setSelectedWallets] = useState(configWallets || []);
+  const { blockchains } = useMetaStore.use.meta();
+  const onChangeWallets = useConfigStore.use.onChangeWallets();
+  const filteredList = searchValue
+    ? allWallets.filter((w) =>
+        w.title.toLowerCase().includes(searchValue.toLowerCase())
+      )
+    : allWallets;
+  const walletsList =
+    category === 'ALL'
+      ? filteredList
+      : filteredList.filter((wallet) => wallet.networks.includes(category));
+  const onChangeWallet = (wallet: WalletTypes) => {
+    const list = [...selectedWallets];
+    const isAlreadyExist =
```

Comment:
You could rewrite it like this:

`const isAlreadyExist = list.includes(wallet);`

Reply:
fixed. I had used previous logic.